### ISSUE TITLE
'AttributeError: module 'taichi' has no attribute 'dynamics''.

### ISSUE THREAD

**Issue by bobcao3** [yuanming-hu/taichi_mpm#55]
https://github.com/taichi-dev/taichi/issues/4364

**Comment by k-ye** [comment]
I don't think this repo is compatible with the new Taichi programming language (maybe even starting from v0.6.0), is it?

**Comment by BoyuanTang331** [comment]
hi， from 0.7.20 there is no more 'ti.dynamics'
from the newest pip, the oldest version I can find is 0.7.20

![2I6YTPH2J%BYA3XG@B)ZWB9](https://github.com/yuanming-hu/taichi_mpm/assets/117408630/5caa0abc-b600-4473-86f5-6788b5f5fc3e)

![image](https://github.com/yuanming-hu/taichi_mpm/assets/117408630/63da1d45-b364-4d4d-a70e-8c7abd8f4efc)
